### DISCUSSION TITLE
Web instance is facing an issue: faker is not defined

### DISCUSSION THREAD

**Discussion by JayantGoel001** [lowlighter/metrics#1120] [General]
Hello @lowlighter,

![image](https://user-images.githubusercontent.com/54479676/177007610-90619f2f-4ed2-449a-8e56-ed2c71e10169.png)

**Comment by lowlighter** [comment]
Hi 👋 !

Can you try reloading the website after cleaning the cache ?
I suspect that because of the change to `faker@v7` and the `<script>` tag that didn't add a new query param to force the reload of it, your browser still use the old cached version of `faker@v5`

**Reply by JayantGoel001** [reply]
Yup, it worked👍
Thanks 😅
